### DISCUSSION TITLE
DepthVis vs DepthPlanner

### DISCUSSION THREAD

**Discussion by guymelamedSC** [microsoft/AirSim#4050] [Support Q&A]
So according to documentation both depth requests (DepthVis and DepthPlanner) outputs "answer" about depth as distance from camera plane.

(Relevant link: https://github.com/Microsoft/AirSim/blob/master/docs/image_apis.md#what-does-pixel-values-mean-in-different-image-types)

Begs the question, what's the difference?
I seem to be getting similar results for each case (as there is no updated guide to get depth values for each request, I did what seemed to work).
And if the difference is only in some factor, why use one over the other?

(P.S, I'm with UE4 and working in Python ComputerVision mode)

**Comment by merkesim** [comment]
Hi, did you find any solution ?
